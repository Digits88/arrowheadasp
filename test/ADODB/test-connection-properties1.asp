<!--#include file="../config/ADODB.inc"-->
Here is a test of connection properties.<br>
<%
    Set CN = OpenConnection()

    for i = 0 to CN.Properties.Count - 1
        %>CN.Property("<%=CN.Properties(i).Name%>")[<%
        %><%=CN.Properties(i).Type%>]="<%=CN.Properties(i).Value%>"<br>
        <%
    next

    CN.Close
%>

The End.
