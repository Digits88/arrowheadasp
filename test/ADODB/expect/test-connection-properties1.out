
Here is a test of connection properties.<br>
CN.Property("Current Catalog")[8]="Wick2"<br>
        CN.Property("Reset Datasource")[3]=""<br>
        CN.Property("Active Sessions")[3]="0"<br>
        CN.Property("Asynchable Commit")[11]="False"<br>
        CN.Property("Catalog Location")[3]="1"<br>
        CN.Property("Catalog Term")[8]="database"<br>
        CN.Property("Catalog Usage")[3]="7"<br>
        CN.Property("Column Definition")[3]="1"<br>
        CN.Property("NULL Concatenation Behavior")[3]="0"<br>
        CN.Property("Data Source Name")[8]="Wick2"<br>
        CN.Property("Read-Only Data Source")[11]="False"<br>
        CN.Property("DBMS Name")[8]="Microsoft SQL Server"<br>
        CN.Property("DBMS Version")[8]="07.00.1063"<br>
        CN.Property("GROUP BY Support")[3]="2"<br>
        CN.Property("Heterogeneous Table Support")[3]="0"<br>
        CN.Property("Identifier Case Sensitivity")[3]="8"<br>
        CN.Property("Maximum Index Size")[3]="900"<br>
        CN.Property("Maximum Row Size")[3]="8060"<br>
        CN.Property("Maximum Row Size Includes BLOB")[11]="False"<br>
        CN.Property("Maximum Tables in SELECT")[3]="32"<br>
        CN.Property("Multiple Storage Objects")[11]="False"<br>
        CN.Property("Multi-Table Update")[11]="False"<br>
        CN.Property("NULL Collation Order")[3]="4"<br>
        CN.Property("OLE Object Support")[3]="1"<br>
        CN.Property("ORDER BY Columns in Select List")[11]="False"<br>
        CN.Property("Prepare Abort Behavior")[3]="2"<br>
        CN.Property("Prepare Commit Behavior")[3]="2"<br>
        CN.Property("Procedure Term")[8]="stored procedure"<br>
        CN.Property("Provider Name")[8]="MSDASQL.DLL"<br>
        CN.Property("OLE DB Version")[8]="02.00"<br>
        CN.Property("Provider Version")[8]="02.50.4403.6"<br>
        CN.Property("Quoted Identifier Sensitivity")[3]="8"<br>
        CN.Property("Schema Term")[8]="owner"<br>
        CN.Property("Schema Usage")[3]="31"<br>
        CN.Property("SQL Support")[3]="267"<br>
        CN.Property("Structured Storage")[3]="1"<br>
        CN.Property("Subquery Support")[3]="31"<br>
        CN.Property("Isolation Levels")[3]="1118464"<br>
        CN.Property("Isolation Retention")[3]="0"<br>
        CN.Property("Table Term")[8]="table"<br>
        CN.Property("User Name")[8]="test"<br>
        CN.Property("Pass By Ref Accessors")[11]="True"<br>
        CN.Property("Transaction DDL")[3]="8"<br>
        CN.Property("Asynchable Abort")[11]="False"<br>
        CN.Property("Data Source Object Threading Model")[3]="1"<br>
        CN.Property("Output Parameter Availability")[3]="4"<br>
        CN.Property("Persistent ID Type")[3]="4"<br>
        CN.Property("Multiple Parameter Sets")[11]="True"<br>
        CN.Property("Rowset Conversions on Command")[11]="True"<br>
        CN.Property("Multiple Results")[3]="1"<br>
        CN.Property("Provider Friendly Name")[8]="Microsoft OLE DB Provider for ODBC Drivers"<br>
        CN.Property("Connection Status")[3]="1"<br>
        CN.Property("Server Name")[8]="GRENDELB"<br>
        CN.Property("Open Rowset Support")[3]="0"<br>
        CN.Property("Accessible Procedures")[11]="True"<br>
        CN.Property("Accessible Tables")[11]="True"<br>
        CN.Property("Integrity Enhancement Facility")[11]="True"<br>
        CN.Property("Outer Join Capabilities")[3]="127"<br>
        CN.Property("Stored Procedures")[11]="True"<br>
        CN.Property("Driver Name")[8]="SQLSRV32.DLL"<br>
        CN.Property("Driver Version")[8]="03.70.1063"<br>
        CN.Property("Driver ODBC Version")[8]="03.51"<br>
        CN.Property("Like Escape Clause")[8]="Y"<br>
        CN.Property("Special Characters")[8]="#$¿¡¬√ƒ≈∆«»… ÀÃÕŒœ–—“”‘’÷ÿŸ⁄€‹›ﬁﬂ‡·‚„‰ÂÊÁËÈÍÎÏÌÓÔÒÚÛÙıˆ¯˘˙˚¸˝˛ˇ"<br>
        CN.Property("Max Columns in Group By")[3]="0"<br>
        CN.Property("Max Columns in Index")[3]="16"<br>
        CN.Property("Max Columns in Order By")[3]="0"<br>
        CN.Property("Max Columns in Select")[3]="4096"<br>
        CN.Property("Max Columns in Table")[3]="1024"<br>
        CN.Property("Numeric Functions")[3]="16777215"<br>
        CN.Property("SQL Grammar Support")[3]="1"<br>
        CN.Property("Outer Joins")[8]="Y"<br>
        CN.Property("String Functions")[3]="5242879"<br>
        CN.Property("System Functions")[3]="7"<br>
        CN.Property("Time/Date Functions")[3]="2097151"<br>
        CN.Property("File Usage")[3]="0"<br>
        CN.Property("Active Statements")[3]="1"<br>
        CN.Property("Password")[8]="sqlworks"<br>
        CN.Property("Persist Security Info")[11]=""<br>
        CN.Property("User ID")[8]="IUSR_SQL"<br>
        CN.Property("Data Source")[8]=""<br>
        CN.Property("Window Handle")[3]=""<br>
        CN.Property("Location")[8]=""<br>
        CN.Property("Mode")[3]=""<br>
        CN.Property("Prompt")[2]="4"<br>
        CN.Property("Connect Timeout")[3]="15"<br>
        CN.Property("Extended Properties")[8]="^.+$"<br>
        CN.Property("Locale Identifier")[3]="1033"<br>
        CN.Property("Initial Catalog")[8]=""<br>
        CN.Property("OLE DB Services")[3]="-7"<br>
        CN.Property("General Timeout")[3]=""<br>
        CN.Property("Autocommit Isolation Levels")[3]="4096"<br>
        

The End.
